Junk4Dollars-appUITests/Junk4Dollars_appUITests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+
+class Junk4Dollars_appUITests: XCTestCase {
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launch()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testTableCellsExist() {
+        let tablesQuery = XCUIApplication().tables
+        XCTAssertTrue(tablesQuery.staticTexts["Throne of Eldraine Booster Box"].exists)
+        XCTAssertTrue(tablesQuery.staticTexts["Oko, Theif of Crowns"].exists)
+        XCTAssertTrue(tablesQuery.staticTexts["MTG M20 Booster Box"].exists)
+    }
+
+    func testLaunchPerformance() {
```

Comment:
I don't think this test is adding a lot of value right now and it seems easy to add if we want to worry about performance later on.